main/main.c

```diff
@@ -830,16 +830,35 @@ PHPAPI ZEND_COLD void php_log_err_with_severity(const char *log_message, int sys
 			char *tmp;
 			size_t len;
 			zend_string *error_time_str;
+			time_t error_time = 0;
+			suseconds_t error_time_usec = 0;
+
+			const char *error_log_format = "d-M-Y H:i:s e";
+			size_t error_log_format_len = sizeof("d-M-Y H:i:s e") - 1;
```

Comment:
it would be probably better to define it uninitialized and assign in the else condition...

Reply:
moved this initialisation down